deb/contents.go

```diff
@@ -19,7 +19,13 @@ type ContentsIndex struct {
 
 // NewContentsIndex creates empty ContentsIndex
 func NewContentsIndex(db database.Storage, repo PublishedRepo, component string, architecture string, udeb bool) *ContentsIndex {
-	return &ContentsIndex{db: db, repo: repo, component: component, architecture: architecture}
+	index := &ContentsIndex{db: db, repo: repo, component: component, architecture: architecture}
+
+	// clean up old values
+	key := index.Key("", "")
+	db.DeleteByPrefix(key)
```

Comment:
Would this not delete the whole database? (Maybe my understanding of leveldb is incorrect here...)